llvm/lib/CodeGen/VirtRegMap.cpp

```diff
@@ -532,6 +535,26 @@ bool VirtRegRewriter::subRegLiveThrough(const MachineInstr &MI,
   return false;
 }
 
+/// Check if we need to maintain liveness for undef subregister lanes that are
+/// live out of a block.
+bool VirtRegRewriter::needLiveOutUndefSubregDef(const LiveInterval &LI,
```

Comment:
I assume that LI is supposed to hold the main live-interval for SubReg, but it would be good to call it out in the comment.
Also, what is PhysReg used here?